src/pages/markets/instruments/_european-indices.tsx

```diff
@@ -1,13 +1,28 @@
 import React from 'react'
 import Symbol from '../components/helper/_symbol'
 import { stocks_european_indices } from '../static/content/_market-symbols'
+import { EU, ROW } from 'components/containers/visibility'
 
+const stocks_european_indices_eu_endpoint = stocks_european_indices.filter((cfds) => {
+    return cfds?.eu === true
+})
 const EuropeanIndices = () => {
     return (
         <>
-            {stocks_european_indices.map((symbol, index) => (
-                <Symbol key={index} src={symbol.src} text={symbol.text} />
-            ))}
+            <ROW>
+                <>
+                    {stocks_european_indices.map((symbol, index) => (
+                        <Symbol key={index} src={symbol.src} text={symbol.text} />
+                    ))}
+                </>
+            </ROW>
+            <EU>
+                <>
+                    {stocks_european_indices_eu_endpoint.map((symbol, index) => (
+                        <Symbol key={index} src={symbol.src} text={symbol.text} eu={symbol.eu} />
```

Comment:
Using map indices as key props is anti-pattern, please check [this ](https://medium.com/wesionary-team/using-index-as-a-key-is-an-anti-pattern-in-react-8e5db3aea3a9) out.